modules/osd-aws-privatelink-firewall-prerequisites.adoc

```diff
@@ -202,14 +202,16 @@ Alternatively, if you wish to not use a wildcard for Amazon Web Services (AWS) A
 |443
 |This alerting service is used by the in-cluster alertmanager to send alerts notifying Red Hat SRE of an event to take action on.
 
+ifdef::openshift-dedicated[]
```

Comment:
The only active includes for this module look to be in ROSA. Searching in the topic maps for the two assemblies that include this module returns the following:

```
pneedle@laptop openshift-docs (OSDOCS-3253)$ grep -is 'rosa-aws-prereqs\|rosa-sts-aws-prereqs' _topic_maps/*
_topic_maps/_topic_map_rosa.yml:  File: rosa-sts-aws-prereqs
_topic_maps/_topic_map_rosa.yml:    File: rosa-aws-prereqs
```

Is my assumption correct that this `ifdef` conditional is being put in place instead of removing these lines, just to future-proof the module in case it is later added to the OSD library? Or, am I missing where this is previewed for OSD? Please feel free to ignore this comment, because the content is successfully removed from the ROSA assembly in the preview.

Reply:
I just put it there as a safety measure in the event that we do share the content.